tajo-cli/src/main/java/org/apache/tajo/cli/tsql/commands/ExitCommand.java

```diff
@@ -37,6 +37,7 @@ public String getCommand() {
   @Override
   public void invoke(String[] cmd) throws Exception {
     context.getOutput().println("bye!");
+    context.getOutput().flush();
```

Comment:
```flush()``` flushes content of buffer to destination. And you can write something again into the stream. ```close()``` flushes content to destination and closes the stream. After execute the command for terminating TSql, we will never write anything anymore. Thus, we need to use ```close()``` instead of  ```flush()```.